tests/cypress/integration/canvas3d_functionality/case_56_canvas3d_functionality_basic_actions.js

```diff
@@ -0,0 +1,158 @@
+        cy.openTaskJob(taskName);
+    });
+
+    after(() => {
+        cy.goToTaskList();
+        cy.deleteTask(taskName);
+    });
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('Check existing of elements.', () => {
+            cy.get('.cvat-canvas3d-perspective')
+                .should('exist')
+                .and('be.visible')
+                .within(() => {
+                    cy.get('.cvat-canvas3d-perspective-arrow-directions')
+                        .should('exist')
+                        .and('be.visible')
+                        .within(() => {
+                            cy.get('[aria-label="arrow-up"]').should('exist').and('be.visible');
+                            cy.get('[aria-label="arrow-left"]').should('exist').and('be.visible');
+                            cy.get('[aria-label="arrow-down"]').should('exist').and('be.visible');
+                            cy.get('[aria-label="arrow-right"]').should('exist').and('be.visible');
+                        });
+                    cy.get('.cvat-canvas3d-perspective-directions')
+                        .should('exist')
+                        .and('be.visible')
+                        .within(() => {
+                            cy.contains('button', 'U').should('exist').and('be.visible');
+                            cy.contains('button', 'I').should('exist').and('be.visible');
+                            cy.contains('button', 'O').should('exist').and('be.visible');
+                            cy.contains('button', 'J').should('exist').and('be.visible');
+                            cy.contains('button', 'K').should('exist').and('be.visible');
+                            cy.contains('button', 'L').should('exist').and('be.visible');
+                        });
+                });
+            cy.get('.cvat-canvas3d-topview').should('exist').and('be.visible');
+            cy.get('.cvat-canvas3d-sideview').should('exist').and('be.visible');
+            cy.get('.cvat-canvas3d-frontview').should('exist').and('be.visible');
+            cy.get('.cvat-canvas-controls-sidebar')
+                .should('exist')
+                .and('be.visible')
+                .within(() => {
+                    cy.get('.cvat-move-control').should('exist').and('be.visible');
+                    cy.get('.cvat-cursor-control').should('exist').and('be.visible');
+                    cy.get('.cvat-draw-cuboid-control').should('exist').and('be.visible');
+                    cy.get('[aria-label="camera"]').should('exist').and('be.visible');
+                });
+        });
+
+        it('Check workspace selector.', () => {
+            cy.get('.cvat-workspace-selector').should('contain.text', 'Standard 3D').click();
+            cy.get('.cvat-workspace-selector-dropdown')
+                .not('.ant-select-dropdown-hidden')
+                .within(() => {
+                    cy.get('[title="Standard 3D"]').should('have.class', 'ant-select-item-option-active');
```

Comment:
Let's not to use ant classes. We can add classes on our side if necessary. Can you do it or do you need any help?

Reply:
I agree that it is better not to use them. I can spend some time adding additional classes. But I'll probably need some help.

Reply:
To exclude the use of these classes, I reworked the attempts to click on the blocked menu items and check the selector text. It should not change at the same time.